src/widgets/SelectStatus/SelectStatus.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, memo } from 'react';
+import {
+    FormControl, InputLabel, MenuItem, Select, SelectChangeEvent,
+} from '@mui/material';
+import { StatusEnum } from '../../entities/Task';
+import { returnStatusColor } from '../../shared/lib/helpers';
+
+interface SelectStatusProps {
+    onChange?: (e: SelectChangeEvent) => void;
+    value: StatusEnum;
+    color?: 'success' | 'primary' | 'secondary'
+}
+
+export const SelectStatus: FC<SelectStatusProps> = memo(({ onChange, value, color = 'primary' }) => (
+    <FormControl sx={{ m: 1, minWidth: 120 }} size="small">
+        <InputLabel color={returnStatusColor(value)}>Status</InputLabel>
+        <Select
+            value={value}
+            label="Status"
+            onChange={onChange}
+            onClick={(event) => event.stopPropagation()}
+            color={returnStatusColor(value)}
```

Comment:
call `returnStatusColor(value)` only once and store result in a const

Reply:
Fixed